transaction-service/src/main/java/pl/adamboguszewski/transaction/service/infrastructure/controller/ApplicationController.java

```diff
@@ -1,17 +1,73 @@
 package pl.adamboguszewski.transaction.service.infrastructure.controller;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 import pl.adamboguszewski.transaction.service.api.transaction.CreateTransactionRequest;
+import pl.adamboguszewski.transaction.service.api.transaction.CreateTransactionResponse;
+import pl.adamboguszewski.transaction.service.application.Transaction;
+import pl.adamboguszewski.transaction.service.application.TransactionService;
 
 @RestController
-@RequestMapping
+@RequestMapping("/")
 public class ApplicationController {
```

Comment:
Was it always named ApplicationController? lol rename it to TransactionController